lib/mix/tasks/elixir.ex

```diff
@@ -0,0 +1,45 @@
+defmodule Mix.Tasks.Thrifter.Elixir do
+  use Mix.Task
+
+  @shortdoc "Generate Thrift Elixir client"
+
+
+  def run(_) do
+    {src_prj_name, gen_prj_name} = generate_project()
+    compile_thrift(gen_prj_name)
+    Thrifter.TemplateManager.instantiate_templates(src_prj_name, gen_prj_name)
+    compile_elixir(gen_prj_name)
+  end
```

Comment:
I think it is a very good comment. 👍 

Let's aim for readability over short but confusing names.